janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -354,7 +359,18 @@ public void mutate(Map<String, Map<String, IndexMutation>> mutations, KeyInforma
                                     fieldDeletions.remove(indexEntry);
                                 }
                             }
-                            deleteIndividualFieldsFromIndex(collectionName, keyIdField, docId, fieldDeletions);
```

Comment:
Is it possible to keep it in the deleteIndividualFieldsFromIndex() private method? Like you have the buildFieldAdditions().